### PR TITLE
Merge master into dev-1.21 

### PR DESCRIPTION
**Pull request by PI-Victor**
Merge master into dev-1.21 to keep in sync - SIG-Release 1.21 Docs team 3/26/21

/cc @reylejano @mvortizr @ChandaniM123 @tengqm

### CODE REVIEW

File: content/en/docs/reference/access-authn-authz/admission-controllers.md [hunk 1]
```diff
@@ -193,7 +193,7 @@ This admission controller will deny exec and attach commands to pods that run wi
 allow host access.  This includes pods that run as privileged, have access to the host IPC namespace, and
 have access to the host PID namespace.
 
-The DenyEscalatingExec admission plugin is deprecated and will be removed in v1.21.
+The DenyEscalatingExec admission plugin is deprecated.
```
Comment (PI-Victor) [inline content/en/docs/reference/access-authn-authz/admission-controllers.md:196]: @reylejano there were conflicts between master and dev-1.21, i've chosen to fix them by following your recent change from master here.

File: content/en/docs/reference/access-authn-authz/admission-controllers.md [hunk 2]
```diff
@@ -206,7 +206,7 @@ is recommended instead.
 This admission controller will intercept all requests to exec a command in a pod if that pod has a privileged container.
 
 This functionality has been merged into [DenyEscalatingExec](#denyescalatingexec).
-The DenyExecOnPrivileged admission plugin is deprecated and will be removed in v1.21.
+The DenyExecOnPrivileged admission plugin is deprecated.
```
Comment (PI-Victor) [inline content/en/docs/reference/access-authn-authz/admission-controllers.md:209]: @reylejano same comment here

**Comment by kbhawkey** [comment]
Thanks @PI-Victor .
/approve

**Pull request merged**